docs/uma-tokenholders/guidence-on-collateral-currency-addition.md

```diff
@@ -0,0 +1,35 @@
+### Guidance for the Addition of Collateral Currencies
+
+Collateral currencies can be added through the collateral currency UMIP template available on Github, by adding 
+ - the name of the token to the be added 
+ - the contract address of the token  together with a link to the address on etherscan
+ - the amount that should be added to the store as final fee expressed in units of the intended collateral currency.   This should equate to approximately $400 at the time the UMIP is submitted.
```

Comment:
For some of the funky coins that aren't available on price feeds like coinmarketcap, it would be nice for them to provide their calculation on how they determined the final fee.